src/main/java/cpw/mods/modlauncher/ClassTransformer.java

```diff
@@ -121,7 +121,8 @@
         finalFlags &= ~ILaunchPluginService.ComputeFlags.SIMPLE_REWRITE; //Strip any modlauncher-custom fields
 
         //Only use the TransformerClassWriter when needed as it's slower, and only COMPUTE_FRAMES calls getCommonSuperClass
-        ClassWriter cw = finalFlags == ILaunchPluginService.ComputeFlags.COMPUTE_FRAMES ? new TransformerClassWriter(this, clazz) : new ClassWriter(finalFlags | Opcodes.ASM7);
+        boolean requireFrames = (finalFlags & ILaunchPluginService.ComputeFlags.COMPUTE_FRAMES) == ILaunchPluginService.ComputeFlags.COMPUTE_FRAMES;
```

Comment:
For brevity you can use `(flagField & MASK) != 0` rather than `(flagField & MASK) == MASK` when `MASK` is a single bit.